patterns/base/table/table.twig

```diff
@@ -0,0 +1,18 @@
+<table class="{{table.class}}">
+  <thead class="{{table.head.class}}">
+    <tr class="base--tr">
```

Comment:
hey @scottnath  - just curious - why should these be separate if they will always be a unit?